docs/tutorial/getting-started.md

```diff
@@ -1,3 +1,215 @@
 # Getting started
 
-TODO
+In this tutorial, we will download and install Pebble, configure layers, run the Pebble daemon, and work with layers and services. After this tutorial, you will have a basic understanding of what Pebble is, how to install it, and how to use it to orchestrate services. It takes about 15 minutes to complete.
+
+
+After that, you can continue exploring more advanced features and use cases (links at the end).
+
+## Prerequisites
+
+- A Linux machine.
+
+## Download and install Pebble
+
+Find the latest tag on the [latest release page](https://github.com/canonical/pebble/releases/latest), then run the following commands to download, extract, and install the latest release (replace `v1.11.0` with the latest tag and `amd64` with your architecture):
```

Comment:
```suggestion
Get the latest tag on the [latest release page](https://github.com/canonical/pebble/releases/latest), then run the following commands to download, extract, and install the latest Pebble release. Replace `v1.12.0` with the latest tag and `amd64` with your architecture, and make sure that {file}`/usr/local/bin/` is included in your system `PATH` variable.
```

Reply:
Find/Get - what's the difference? If we have to get down to the word: I'm not a native speaker, but for me, the process is more "find" than "get" - you open the page, you have to find where the tag is. It's not like it's a CURL command where you run it and you immediately get the tag alone in the result.

1.11 -> 1.12: updated.

release -> pebble release: redundant wording. This is the release page for pebble, this is the tutorial for pebble, and most importantly, this is the doc site for pebble. Why bombarding the users with pebble whenever possible?

/user/local/bin in path: added.